lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/MethodGenerator.java

```diff
@@ -9,8 +9,8 @@
 import javax.annotation.processing.ProcessingEnvironment;
 import javax.lang.model.element.*;
 import javax.lang.model.type.TypeMirror;
-import javax.lang.model.util.SimpleAnnotationValueVisitor14;
-import javax.lang.model.util.SimpleElementVisitor14;
+import javax.lang.model.util.SimpleAnnotationValueVisitor9;
```

Comment:
Does this mean that at the end, we should compile our sources to `--target=9`?

Reply:
Target 11 seems to be the safest: https://github.com/enso-org/enso/pull/8094#discussion_r1362356822